packages/equation_extraction/Dockerfile

```diff
@@ -63,30 +53,35 @@ RUN mv /taskrunner/build/libs/*.jar .
 RUN mv /terarium-1.0.0-SNAPSHOT.jar /taskrunner.jar
 
 # ------------------------------------------------------------------------------
-
-# Set up the Python image with JRE
-FROM python:3.10-slim
+FROM python:3.11-slim
```

Comment:
3.10 or 3.11???

Reply:
Who knows! I just bumped because why not